textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/fuzzytokenmatcher/FuzzyTokenMatcherTest.java

```diff
@@ -151,8 +151,8 @@ public void TestFuzzyTokenMatcherWithLargeTokens() throws Exception {
         
         List<ITuple> results = getQueryResults(query, threshold, attributeList, isSpanInformationAdded);
         boolean contains = TestUtils.containsAllResults(expectedResultList, results);
-        //Assert.assertTrue(contains);
-        //Assert.assertEquals(3,results.size());
+        Assert.assertTrue(contains);
+        Assert.assertEquals(3,results.size());
```

Comment:
If we already did `containsAllResults()`, why do we still need this `assertEquals()`?